### PR TITLE
Fix par bleeding

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
As described in #391 

So now we add a new list of original par (as defined on the loading of the package) to our `gsconfig` env , and add that to the custom gsconfig$options when `loadConfig` is called. I therefore had to change some tests. 

@jread-usgs @jiwalker-usgs and/or @lindsaycarr have a look.

### CODE REVIEW

File: R/config.R [hunk 1]
```diff
@@ -1,10 +1,12 @@
 gsconfig <- new.env(parent = emptyenv())
+gsconfig$orignial.par <- par(no.readonly = TRUE)
```
Comment (lindsayplatt) [inline R/config.R]: just so this doesn't mix us up later - the n and the i are mixed up here: original

Comment (ldecicco-USGS) [inline R/config.R]: that sounds like my coding!

File: R/config.R [hunk 2]
```diff
@@ -21,10 +23,14 @@ loadConfig = function(filename) {
   }
 
   graphTemplate <- yaml.load_file(filename)
-
+  origPar <- gsconfig$orignial.par
```
Comment (lindsayplatt) [inline R/config.R]: spelling here too

**Comment by lindsayplatt** [comment]
in the updated figs, some of the stuff looks like it's getting cutoff now

File: R/config.R (hunk 1 above)
Comment (ldecicco-USGS) [inline R/config.R]: I'm also wondering if we want to do a dev.new right before catching the par to truly get R's defaults, or is there some off-chance that someone will have set par assume it's going to persist?

File: R/config.R (hunk 2 above)
Comment (ldecicco-USGS) [inline R/config.R]: well, if it was only misspelled in one location, it wouldn't work

**Comment by ldecicco-USGS** [comment]
I took out my changes, rebuilt the Readme, and the newly created figures (for the readme) look the same as with my changes I'm proposing here (ie...these changes aren't causing the altered figures). @lindsaycarr could you verify that your fork's readme (assuming it's current with USGS-R) produces the graphs without changes? If you don't see altered figures, could you pull this PR locally and see if you produce the same readme figures that I now?

What's weird and interesting is that the graphs I'm pushing here look fine when done in the R console...but different when created via knitr. So that may be a separate issue.

**Comment by lindsayplatt** [comment]
So when I knit the README on my local version (updated with most recent upstream), none of the files changed. I pulled down your branch, and knit without building and reloading and the figures changed but back to how I would expect them to look. Then I built and reloaded and none of the figures show up in the Git tab, but they look different than I would expect them to.

**Comment by ldecicco-USGS** [comment]
huh? "Then I built and reloaded and none of the figures show up in the Git tab, but they look different than I would expect them to."

**Pull request merged**